buildSrc/version.properties

```diff
@@ -4,7 +4,8 @@ lucene            = 6.1.0
 # optional dependencies
 spatial4j         = 0.6
 jts               = 1.13
-jackson           = 2.7.1
+jackson           = 2.8.0.rc2
```

Comment:
Yes, I commented the pull request with "This is work in progress"

I'm just tracking all necessary changes here so that I have some code to share with Jackson's team.